scenario-examples-bootstrapper/setup/nu/import-and-deploy-example-scenarios.sh

```diff
@@ -14,13 +14,19 @@ SCENARIO_EXAMPLE_DIR_PATH=${1%/}
 EXAMPLE_SCENARIO_NAME=$2
 EXAMPLE_SCENARIO_FILE=$3
 
-echo "Starting to import and deploy example scenarios..."
+echo "Starting to import and deploy example scenario..."
```

Comment:
file name can also be renamed - scenarios -> scenario

Reply:
Good idea, fixed